src/base/net/stratum/SelfSelectClient.cpp

```diff
@@ -235,6 +240,50 @@ void xmrig::SelfSelectClient::submitBlockTemplate(rapidjson::Value &result)
     });
 }
 
+int64_t xmrig::SelfSelectClient::submit(const JobResult& result)
+{
+    if (m_submit_to_origin) {
+        submitOriginDaemon(result);
+    }
+    return m_client->submit(result);
+}
+
+void xmrig::SelfSelectClient::submitOriginDaemon(const JobResult& result)
+{
+    if (result.diff == 0 || m_targetdiff == 0) {
+        return;
+    }
+    
+    if (result.actualDiff() < m_targetdiff) {
+        m_origin_not_submitted++;
+        LOG_INFO("%s " RED_BOLD("not submitted to origin daemon, difficulty too low") " (%" PRId64 "/%" PRId64 ") "
```

Comment:
Surely this is going to spam the origin? Most job submits will be valid at job difficulty but below block difficulty.

Reply:
Thus I suggest either removing this logging or at least changing here to LOG_DEBUG.

Reply:
Hi @jtgrassie, thank you for reviewing!

The thinking here was to display an equal amount of messages about submitting to the origin daemon than to the pool, so a user can see the stats at first glance. As observed, with pool difficulties generally low, most jobs submitted there will not be submitted to the origin daemon.

Will change the log level as suggested.